### PR TITLE
docs(readme): add documentation and minor fixes

### PR DESCRIPTION
**Pull request by KobeW50**
closes https://github.com/ReVanced/revanced-manager/issues/962

### CODE REVIEW

**Comment by Ushie** [comment]
Please rebase your PR ontop of the latest dev commit

**Comment by oSumAtrIX** [comment]
Apparently is already.

File: README.md [hunk 1]
```diff
@@ -3,30 +3,32 @@
 The official ReVanced Manager based on Flutter.
 
 ## 🔽 Download
-
-You can obtain ReVanced Manager by downloading it from either [revanced.app/download](https://revanced.app/download) or [GitHub Releases](https://github.com/ReVanced/revanced-manager/releases)
+To download ReVanced Manager, go [here](https://github.com/revanced/revanced-manager/releases/latest) and install the provided APK file.
```
Comment (oSumAtrIX) [inline README.md:6]: "Go here" is weird.

File: README.md [hunk 2]
```diff
@@ -3,30 +3,32 @@
...
 
 ## 📝 Prerequisites
-
 1. Android 8 or higher
-2. Incompatible with certain ARMv7 devices
+2. Does not work on some armv7 devices
```
Comment (oSumAtrIX) [inline README.md:10]:
```suggestion
2. Does not work on some ARMv7 devices
```

File: README.md [hunk 3]
```diff
@@ -3,30 +3,32 @@
...
-## 🔴 Issues
+## 📃 Documentation
+The documentation can be found [here](https://github.com/revanced/revanced-manager/tree/main/docs).
 
+## 🔴 Issues
 For suggestions and bug reports, open an issue [here](https://github.com/revanced/revanced-manager/issues/new/choose).
 
-## 🌐 Translation
+## 💭 Discussion
+If you wish to discuss the Manager, a thread has been made under the [#development](https://discord.com/channels/952946952348270622/1002922226443632761) channel in the Discord server. Please note that this thread may be temporary and may be removed in the future.
```
Comment (oSumAtrIX) [inline README.md:19]: This should be removed as staged in dev.

**Comment by Ushie** [comment]
It reverts changes of d1218616ec870edbaced6fd19dd6a2a779c73968

File: README.md [hunk 4]
```diff
@@ -3,30 +3,32 @@
...
+
 
+## 🌐 Translation
 [![Crowdin](https://badges.crowdin.net/revanced/localized.svg)](https://crowdin.com/project/revanced)
 
-We're accepting translations on [Crowdin](https://translate.revanced.app)
+If you wish to translate ReVanced Manager, we're accepting translations on [Crowdin](https://translate.revanced.app)
```
Comment (oSumAtrIX) [inline README.md:25]: It is unnecessary to question if you wish to translate.

```suggestion
We're accepting translations on [Crowdin](https://translate.revanced.app)
```

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

**Comment by oSumAtrIX** [comment]
> It reverts changes of [d121861](https://github.com/ReVanced/revanced-manager/commit/d1218616ec870edbaced6fd19dd6a2a779c73968)

Well, thats what they PR'd. Not sure about their intention though.

**Comment by Ushie** [comment]
Their changes are based on main, but target dev

**Comment by oSumAtrIX** [comment]
The PR source branch is based on `dev` with a reverting commit on top.

**Comment by KobeW50** [comment]
Im a dumbass. I must've edited the main branch and then put it on the dev

**Comment by oSumAtrIX** [comment]
Simply drop the last commit and commit a new one.

**Comment by KobeW50** [comment]
I dont use git so i couldn't figure how to drop a commit. But the current file changes should be good now

File: README.md [hunk 5]
```diff
@@ -19,14 +22,14 @@ For suggestions and bug reports, open an issue [here](https://github.com/revance
...
 
-We're accepting translations on [Crowdin](https://translate.revanced.app)
+We're accepting translations on [Crowdin](https://translate.revanced.app).
 
 ## 🛠️ Building Manager from source
 
 1. Setup flutter environment for your [platform](https://docs.flutter.dev/get-started/install)
 2. Clone the repository locally
-3. Add your github token in gradle.properties like [this](/docs/4_building.md)
+3. Add your GitHub token in gradle. Properties like [this](/docs/4_building.md)
```
Comment (validcube) [inline README.md:31]: ❗Should be called `gradle.properties`, not `gradle. Properties`

```suggestion
3. Add your GitHub token in gradle.properties like [this](/docs/4_building.md)
```

**Review by validcube — CHANGES_REQUESTED** [review]
Bloop

**Review by validcube — APPROVED** [review]
🥞 Changes to the readme file been approved.

Thank you for your contribution!

**Comment by KobeW50** [comment]
Thank you @validcube ❤️

**Pull request merged**